eng/pipelines/libraries/base-job.yml

```diff
@@ -109,7 +109,12 @@ jobs:
           - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
             - _stripSymbolsArg: -stripSymbols
 
-        - _buildArguments: -configuration ${{ parameters.buildConfig }} -ci -arch ${{ parameters.archType }} $(_finalFrameworkArg) $(_stripSymbolsArg) $(_testScopeArg) $(_warnAsErrorArg) $(_runtimeOSArg) $(_msbuildCommonParameters) $(_runtimeArtifactsPathArg) $(_crossBuildPropertyArg)
+        # force a value for OS when cross-building iOS on OSX
+        - ${{ if eq(parameters.osGroup, 'iOS') }}:
+          - name: _osOverride
```

Comment:
There is already a _runtimeOSArg, why not use that, just like we do with wasm? https://github.com/dotnet/runtime/blob/deb35b83f33c27500940bb0313024f254974ae89/eng/pipelines/libraries/base-job.yml#L71

Reply:
.... oopsie